lib/jt400.ts

```diff
@@ -306,11 +329,20 @@ export interface CLOB {
 
 export type Param = string | number | Date | null | CLOB
 
-export interface DataQOptions {
-	name: string
+
+export interface JustNameMessageQ{
+	name:string
+}
+export interface JustMessageDataQ{
+	path:string
+}
+export type MessageQOptions = JustNameMessageQ | JustMessageDataQ
+
+export interface MessageQReadOptions {
+	wait?: number
 }
 
-export interface DataQReadOptions {
+export interface KeyedDataQReadOptions {
```

Comment:
This would be a breaking change. Do we really need this?

Reply:
Why would it be a breaking change? It's just changing the name of the type interface --- I thought this was more clear because there are also data queues which are not keyed.

Reply:
Yes, it is a better name. But it will produce a compile error for anyone who is importing and using this interface in typescript.

Reply:
OOoooooOOooohhhh. . . that makes a lot of sense.  I will change it back.